java-rmi/pom.xml

```diff
@@ -0,0 +1,23 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.baeldung.rmi</groupId>
+  <artifactId>java-rmi</artifactId>
+  <version>1.0-SNAPSHOT</version>
+  <packaging>jar</packaging>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>3.8.1</version>
```

Comment:
This needs to have the parent module in a <parent> tag. Go ahead and check a couple other poms for examples.

Also, the parent pom will include junit for you, so you can remove this.

Finally, this can't be part of the build unless this module "java-rmi" is listed in the module list in tutorials/pom.xml. Let's add that.